### PR TITLE
Release 1.8: prepare the release notes and upgrade notes

### PR DESCRIPTION
**Pull request by maelvls**
I re-created https://github.com/cert-manager/website/pull/839 due to me pushing to cert-manager/website instead of maelvls/website. I could not force push to the branch on cert-manager/website.

Later on, I will update the release notes with the command:

```sh
GITHUB_TOKEN=$(lpass show -p github.com) START_SHA="$(git rev-list --reverse --ancestry-path $(git merge-base v1.7.0 master)..master | head -1)" END_REV="master" BRANCH="master" RELEASE_VERSION="1.8.0" release-notes --repo-path=cert-manager --org=cert-manager --repo=cert-manager --required-author=jetstack-bot --output /tmp/release-notes-1.8.0.md
```

The reason I am doing this so early is that I wish to review each changelog entry "from the start" and as early as possible, instead of doing it at the last moment.

If I need to reformulate one of the changelog entries, I usually also update the `release-note` block in the original PR description. It is not the best workflow, but at least any subsequent run of release-note will pick up the changes.

@SgtCoDFish I have opened this PR against master this time, since I am not sure release-next still makes sense with the new website.

### CODE REVIEW

**Comment by SgtCoDFish** [comment]
Looks like a good start! I think the release notes I've written for v1.8.0-alpha.0 will be useful here, I spent a bunch of time on them this morning: https://github.com/cert-manager/cert-manager/releases/tag/v1.8.0-alpha.0

Agree that release-next is not useful now!

We might need to merge https://github.com/cert-manager/website/pull/877 before we start working on next-docs (if we're gonna start working on next-docs that is)

**Comment by maelvls** [comment]
@SgtCoDFish I have copy-pasted the release notes you have written in 1.8.0-alpha.0 and 1.8.0-alpha.1. I have gone over each PR and made sure the `release-note` block is easily understandable.

File: content/en/docs/release-notes/release-notes-1.8.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+### Other (Cleanup or Flake)
+
+- Aggregated admin and edit roles will now include permissions to update certificates' status, which will allow namespace admins and editors to run the `cmctl renew` command in their namespaces. ([#4955](https://github.com/cert-manager/cert-manager/pull/4955), [@andreadecorte](https://github.com/andreadecorte))
+- Cleanup: No longer log an error when cert-manager encounters a conflict in the secrets manager, in favour of always force applying. ([#4815](https://github.com/cert-manager/cert-manager/pull/4815), [@JoshVanL](https://github.com/JoshVanL))
+- Failed certificate issuances are now retried with an exponential backoff where the backoff periods are 1h, 2h, 4h, 8h, 16h, 32h. 
+  A new field `failedIssuanceAttempts` is added to `Certificate`'s status that keeps track of consecutive failed issuances. Backoff period gets reset by a successful issuance. The current behaviour where changing certain fields on `Certificate`s spec (such as DNS names) or manually renewing using `cmctl` tool remains unchanged. ([#4772](https://github.com/cert-manager/cert-manager/pull/4772), [@irbekrm](https://github.com/irbekrm))
```
Comment (irbekrm) [inline content/en/docs/release-notes/release-notes-1.8.md:121]: Suggestion: arguably I think this too could go into 'Major themes' or a similar block as 'exponential backoff'. I'd be interested to hear users feedback on how the exponential backoff feature works for them (as on that depends some potential future work https://github.com/cert-manager/cert-manager/issues/4786) so would like for it to be more noticeable

**Review by irbekrm — COMMENTED** [review]
The release notes look awesome!

I came here to add a note on supported kube versions for server-side-apply, but didn't see where to add it, so added two suggestions for a 'themes section' like we did for 1.7 release notes

File: content/en/docs/release-notes/release-notes-1.8.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+- Certificate AdditionalOutputFormat: AdditionalOutputFormats are actively reconciled, meaning cert-manager will always maintain the correct Secret data, both for fields being removed/added/modified on both the Certificate's AdditionalOutputFormat as well as the target Secret's Data. AdditionalOutputFormat is an Alpha feature, and is only enabled with the `--feature-gates=AdditionalCertificateOutputFormats=true` flag. ([#4813](https://github.com/cert-manager/cert-manager/pull/4813), [@JoshVanL](https://github.com/JoshVanL))
+- ClusterRoles aggregation to user-facing admin/edit/view ClusterRoles can be optionally turned off ([#4937](https://github.com/cert-manager/cert-manager/pull/4937), [@illrill](https://github.com/illrill))
+- ServerSideApply: The feature gate `ServerSideApply=true` configures the certificate-shim controllers to use Kubernetes Server Side Apply on Certificate resources. ([#4811](https://github.com/cert-manager/cert-manager/pull/4811), [@JoshVanL](https://github.com/JoshVanL))
```
[the remark this answers is not in the document]
Comment (SgtCoDFish) [reply content/en/docs/release-notes/release-notes-1.8.md:92]: I absolutely think we should do that :+1:

**Comment by irbekrm** [comment]
sorry, broke the spell checker, will fix

**Pull request merged**